wemake_python_styleguide/visitors/ast/complexity/counts.py

```diff
@@ -351,3 +253,26 @@ def _check_yield_values(self, node: ast.Yield) -> None:
                         baseline=constants.MAX_LEN_YIELD_TUPLE,
                     ),
                 )
+
+
+@final
+class TupleUnpackVisitor(BaseNodeVisitor):
+    """Finds statements with too many variables receiving an unpacked tuple."""
+
+    def visit_Assign(self, node: ast.Assign) -> None:
```

Comment:
Are other `Assign` types possible here? Like `AugAssign` or `AnnAssign`?

Reply:
I'm sorry but I don't fully understand. Are you asking whether tuple unpacking is possible with augmented/annotated assignments? If so, I don't think so.